app/views/wizard/steps/import_dates/show.html.erb

```diff
@@ -9,7 +9,7 @@
         <%= f.govuk_date_field :import_date, legend: { text: 'When will the goods be imported?', size: 'xl', tag: 'h1' }, hint: { text: 'As duties and quotas change over time, it may be important to enter the proposed import date. For example, 27  3  2021' } %>
         <%= f.govuk_submit %>
       <% end %>
-      <%= render 'shared/commodity_details' %>
+      <%= render 'shared/commodity_details', locals: { commodity_code: @commodity.code, commodity_description: @commodity.description } %>
```

Comment:
I think `@commodity` should be accessible inside that partial? Not sure if you need to explicitly pass it, as this variable will be available at the controller level.

Reply:
Yeah, I couldn't see obvious idioms on the web beyond the rails docs wanting you to use locals with partials. These partials also have access to params, all the methods in the base controller and all the methods in current controller.